### PR TITLE
Update functionality before deployment

### PR DESCRIPTION
**Pull request by SergePogorelov**
This PR introduces several updates and improvements to the annotation tool:

**Add Instruction Component:**
- Added a new component with instructions for using the app, accessible via an expandable/collapsible button.

**Update Clear Annotation Functionality:**
- Renamed the "Clear Annotation" button to "Start Over".
- The "Start Over" button now clears the current annotation from the map, allowing users to start fresh without saving changes to the database.

**Update generate_prediction Logic:**
- Always sets "Prediction Hexagon IDs" as the starting point for annotations.

**Add Load Annotation Functionality:**
- Introduced the ability to load the last saved annotation for the currently selected taxon.

**Default Map Layer Settings:**
- The "Prediction Hexagons" map layer is now unchecked by default.

**Update Sidebar Component:**  
- Made the Model field read-only for better clarity.

### CODE REVIEW

File: src/frontend/src/components/Instruction.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            <br />
+            <li>
+              <strong className="save-annotation">Save Annotation</strong>: Clicking this button saves the
+              current annotation layer to the database (displaying it in <b style={{ color: 'green' }}>green</b>).
+              <p>Before clicking this button, make sure that the annotation looks
+              correct for the selected Taxa.</p>
+            </li>
+            <br />
+            <li>
+              <strong className="upload-annotation">Upload Annotation</strong>: Clicking this button loads
```
Comment (oz03-hub) [inline src/frontend/src/components/Instruction.js:45]: typo? says load annotation on the button, upload here

File: src/frontend/src/components/Instruction.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+              the last saved annotation for the currently selected Taxa from
+              the database. <p>After loading, you can edit it and save it again. A
+              new save will <b>overwrite</b> the previous one.</p>
+            </li>
+            <br />
+            <li>
+              <strong className="start-over">Start Over</strong>: Clicking this button clears the
```
Comment (oz03-hub) [inline src/frontend/src/components/Instruction.js:52]: this might have to be renamed like we discussed

**Review by oz03-hub — CHANGES_REQUESTED** [review]
a few typos and confusions need to be cleared otherwise all good

File: src/frontend/src/components/Instruction.js (hunk 2 above)
Comment (SergePogorelov) [reply src/frontend/src/components/Instruction.js]: Fixed in commit 3b78378780154fd2d140bf3aed1bdb1a6d3184b8.

File: src/frontend/src/components/Instruction.js (hunk 1 above)
Comment (SergePogorelov) [reply src/frontend/src/components/Instruction.js]: Fixed in commit 3b78378780154fd2d140bf3aed1bdb1a6d3184b8.

**Review by oz03-hub — APPROVED** [review]
Seems like there is a conflict, make sure to resolve carefully while merging

**Review by ginic — COMMENTED** [review]
Overall the changes look good to me. I love having the instructions drop down! 
I did notice an issues with loading annotations, which I put in a comment. Please let me know I've missed something here.

File: src/backend/app/tools.py [hunk 3]
```diff
@@ -69,5 +69,6 @@ def generate_prediction(eval_params):
         pred_loc_combined=pred_loc_combined.tolist(),
         hull_points=hull_points,
         prediction_hexagon_ids=prediction_hexagon_ids,
+        # set prediction_hexagon_ids as the starting point for the annotation
         annotation_hexagon_ids=prediction_hexagon_ids,
     )
```
[the remark this answers is not in the document]
Comment (SergePogorelov) [reply src/backend/app/tools.py:74]: Apparently you clone a version in which I fixed the merge conflict and removed this function.
I will add it in the next PR - there the annotations will be load from the database.

**Review by cheerstopaula — APPROVED** [review]

**Pull request merged**